client/post-editor/editor-more-options/copy-post.jsx

```diff
@@ -58,57 +56,65 @@ class EditorMoreOptionsCopyPost extends Component {
 	}
 
 	goToNewDraft = () => {
-		if ( '' !== this.state.selectedPostId ) {
-			page.redirect( `/post/${ this.props.siteSlug }?copy=${ this.state.selectedPostId }` );
+		const { siteSlug, type } = this.props;
+		const { selectedPostId } = this.state;
+		if ( '' !== selectedPostId ) {
+			page.redirect( `/${ type }/${ siteSlug }?copy=${ selectedPostId }` );
 			this.closeDialog();
 		}
 	}
 
 	render() {
-		const { translate, siteId } = this.props;
-
+		const { siteId, translate, type } = this.props;
+		const { selectedPostId, showDialog } = this.state;
 		const buttons = [ {
 			action: 'cancel',
 			label: translate( 'Cancel' ),
 		}, {
 			action: 'copy',
 			label: translate( 'Overwrite' ),
 			isPrimary: true,
-			disabled: ! this.state.selectedPostId,
+			disabled: ! selectedPostId,
 			onClick: this.goToNewDraft,
 		} ];
 
 		return (
 			<AccordionSection className="editor-more-options__copy-post">
 				<EditorDrawerLabel
-					labelText={ translate( 'Copy Post' ) }
-					helpText={ translate( "Pick a post and we'll copy the title, content, tags and categories. " ) }
+					labelText={ this.isPost() ? translate( 'Copy Post' ) : translate( 'Copy Page' ) }
+					helpText={ this.isPost()
+						? translate( "Pick a post and we'll copy the title, content, tags and categories." )
+						: translate( "Pick a page and we'll copy the title, content, tags and categories." )
+					}
 				>
 					<Button borderless compact onClick={ this.openDialog }>
-						<Gridicon icon="clipboard" /> { translate( 'Select a post to copy' ) }
+						<Gridicon icon="clipboard" />
+						{ this.isPost() ? translate( 'Select a post to copy' ) : translate( 'Select a page to copy' ) }
 					</Button>
 				</EditorDrawerLabel>
 				<Dialog
 					autoFocus={ false }
-					isVisible={ this.state.showDialog }
+					isVisible={ showDialog }
 					buttons={ buttons }
 					onClose={ this.closeDialog }
 					additionalClassNames="editor-more-options__copy-post-select-dialog"
 				>
 					<FormSectionHeading>
-						{ translate( 'Select a post to copy' ) }
+						{ this.isPost() ? translate( 'Select a post to copy' ) : translate( 'Select a page to copy' )
+						}
 					</FormSectionHeading>
 					<p>
-						{ translate( "Pick a post and we'll copy the title, content, tags and categories. " ) }
+						{ '' }
```

Comment:
Ugh, this looks like something I've deleted accidentally or a copypasta gone awry!